src/psd_channel.rs

```diff
@@ -75,42 +75,14 @@ pub trait IntoRgba {
             self.insert_channel_bytes(&mut rgba, TransparencyMask, alpha_channel);
         } else {
             // If there is no transparency data then the image is opaque
-            for idx in 0..rgba_len / 4 {
-                rgba[idx * 4 + 3] = 255;
+            for alpha in rgba.iter_mut().skip(3).step_by(4) {
+                *alpha = 255;
             }
         }
 
         rgba
     }
 
-    /// Generate an RGBA Vec<u8> from a composite image or layer that uses 16 bits per
-    /// pixel. We do this by mapping the 16 bits back down to 8 bits.
-    ///
-    /// The 16 bits are stored across the red and green channels (first and second).
-    fn generate_16_bit_grayscale_rgba(&self) -> Vec<u8> {
```

Comment:
Since we have commented out code hat calls this, don't remove.

Instead, use `#[expect(unused, reason = "...")]`

https://github.com/chinedufn/psd/blob/a57f32fc5d300e3a5398aa313b74cb4461907f69/src/psd_channel.rs#L44-L50

Reply:
`expect` disclaimer is a very new lint, I wrote this before that, and lastly, this will instantly shoot MSRV to 1.81.0 as on older versions its marked experimental